extensions/replication/utils/ObjectQueueEntry.js

```diff
@@ -135,6 +135,24 @@ class ObjectQueueEntry extends ObjectMD {
         return newEntry;
     }
 
+    toPendingEntry(site) {
+        const status = _getGlobalReplicationStatus(this.getValue());
+        return this.clone()
+            .setReplicationSiteStatus(site, 'PENDING')
+            .setReplicationStatus(status);
```

Comment:
Nice catch! Missed this from refactoring this method. 😄

Reply:
Updated so now it will set the global status properly. I think it's worth noting that the global status could still be FAILED in this case if more than one site had a failed status. I don't see a problem with that though because the PENDING status for this site's retry can be seen in the source object's metadata